jerry-core/ecma/builtin-objects/ecma-builtin-number-prototype.c

```diff
@@ -1007,15 +771,21 @@ ecma_builtin_number_prototype_dispatch_routine (uint16_t builtin_routine_id, /**
     }
     case ECMA_NUMBER_PROTOTYPE_TO_FIXED:
     {
-      return ecma_builtin_number_prototype_object_to_fixed (this_arg_number, routine_arg_1);
+      return ecma_builtin_number_prototype_object_to_number_convert (this_arg_number,
+                                                                     routine_arg_1,
+                                                                     NUMBER_ROUTINE_TO_FIXED);
     }
     case ECMA_NUMBER_PROTOTYPE_TO_EXPONENTIAL:
     {
-      return ecma_builtin_number_prototype_object_to_exponential (this_arg_number, routine_arg_1);
+      return ecma_builtin_number_prototype_object_to_number_convert (this_arg_number,
+                                                                     routine_arg_1,
+                                                                     NUMBER_ROUTINE_TO_EXPONENTIAL);
     }
     case ECMA_NUMBER_PROTOTYPE_TO_PRECISION:
```

Comment:
Please merge these 3 cases. the number routine mode can be calculated like this:

NUMBER_ROUTINE_TO_FIXED + (builtin_id - ECMA_NUMBER_PROTOTYPE_TO_FIXED);